ledger/accountdb.go

```diff
@@ -326,9 +323,12 @@ type catchpointState string
 const (
 	// catchpointStateLastCatchpoint is written by a node once a catchpoint label is created for a round
 	catchpointStateLastCatchpoint = catchpointState("lastCatchpoint")
-	// catchpointStateWritingCatchpoint is written by a node while a catchpoint file is being created. It gets deleted once the file
-	// creation is complete, and used as a way to record the fact that we've started generating the catchpoint file for that particular
-	// round.
+	// This state variable is set to 1 if catchpoint's first stage is unfinished,
```

Comment:
please document the `catchpointStateWritingFirstStageInfo` is a persisted analogue of in-memory `dcc.catchpointFirstStage` flag used for recovering